rustls/src/client/client_conn.rs

```diff
@@ -651,7 +651,11 @@ mod connection {
     use crate::suites::ExtractedSecrets;
     use crate::sync::Arc;
 
-    /// Stub that implements io::Write and dispatches to `write_early_data`.
+    /// Allows writing of early data in resumed TLS 1.3 connections
+    ///
+    /// "Early data" is also known as "0-RTT data".
+    ///
+    /// This structure implements [`std::io::Write`].
```

Comment:
Another tiny nit: I think "structure" is a weird fit here, maybe "type"?

Reply:
Added a commit fixing this in the pre-exsiting `ReadEarlyData` struct docs and then matched it in the new `WriteEarlyData` docs.